components/view/nav.tsx

```diff
@@ -77,7 +87,29 @@ export default function Nav({
               )}
             </div>
           </div>
-          <div className="absolute inset-y-0 right-0 flex items-center pr-2 sm:static sm:inset-auto sm:ml-6 sm:pr-0 space-x-2">
+          <div className="absolute inset-y-0 right-0 flex items-center pr-2 sm:static sm:inset-auto sm:ml-6 sm:pr-0 space-x-4">
+            {embeddedLinks && embeddedLinks.length > 0 && (
+              <DropdownMenu>
+                <DropdownMenuTrigger>
+                  <div className="text-sm font-semibold mr-6">
+                    Embedded Links
+                  </div>
+                </DropdownMenuTrigger>
+                <DropdownMenuContent className="space-y-2 pt-2 px-4">
+                  {embeddedLinks.map((link, index) => (
+                    <Link
+                      href={link}
+                      target="_blank"
+                      className="text-sm flex items-start gap-2"
+                      key={index}
+                    >
+                      <span>{index + 1}.</span>
+                      <span className="underline">{link}</span>
+                    </Link>
```

Comment:
You can just use a `<DropdownMenuItem>` from shadcn/ui and the links will be styled properly. 

Please remove the numbering `<span>{index + 1}.</span>`, I don't think that's needed